virttest/qemu_devices/qdevices.py

```diff
@@ -1160,9 +1160,11 @@ def _cmdline_json(self):
                 command_dict[key] = str(val)
             # disable-legacy from device ("driver": "virtio-scsi-pci")
             # write-cache from device ("driver": "scsi-hd")
-            elif val in ('on', 'yes') and key not in expect_string_val:
+            # disable-modern from device ("driver": "virtio-scsi-pci")
+            elif val in ('on', 'yes', "true") and key not in expect_string_val:
                 command_dict[key] = True
-            elif val in ('off', 'no') and key not in expect_string_val:
+            elif val in ('off', 'no', "false") and \
+                    key not in expect_string_val:
```

Comment:
I have a bit of confusion about the code and the description of the commit.
The commit said `qdevices: change type of "disable-modern" to integer ` but didn't see the type of `disable-modern` is changed to interger(`int`), or maybe I misunderstood your code.
Would you like to explain?
Thanks.

Reply:
> I have a bit of confusion about the code and the description of the commit. The commit said `qdevices: change type of "disable-modern" to integer ` but didn't see the type of `disable-modern` is changed to interger(`int`), or maybe I misunderstood your code. Would you like to explain? Thanks.

`disable-modern` is "false" or "true" in string type and it should be converted to boolean.
There would be typo and let me change this typo.
Thanks!